docs/guides/q-ctrl-optimization-solver.ipynb

```diff
@@ -261,7 +275,8 @@
     "maxcut_job = solver.run(\n",
     "    problem=problem_as_str,\n",
     "    problem_type=\"maxcut\",\n",
-    "    instance=hub + \"/\" + group + \"/\" + project,\n",
+    "    instance=instance,  # E.g. \"ibm-q/open/main\"\n",
+    "    backend_name=backend_name,  # E.g. \"ibm_kyiv\"\n",
```

Comment:
This is the main change to most notebooks: We load the `backend_name` and `instance` variables in a hidden cell, then use them in the function call with a comment giving an example of the value.